src/styles/_header.scss

```diff
@@ -37,5 +37,10 @@
     text-transform: uppercase;
     font-size: 12px;
     font-weight: 500;
+    transition-duration: 300ms;
```

Comment:
Need to add rule that you apply transition to.

Reply:
Everything is working but I did not understand rule! Can you explain pls?

Reply:
All is here in the documentation,
You need to specify properties that you transition.
https://developer.mozilla.org/en-US/docs/Web/CSS/transition

Reply:
Not fixed.